crm_phonecall_timesheet/README.rst

```diff
@@ -1,56 +1,91 @@
-.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+=============
+CRM Timesheet
+=============
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Ftimesheet-lightgray.png?logo=github
+    :target: https://github.com/OCA/timesheet/tree/12.0/crm_phonecall_timesheet
+    :alt: OCA/timesheet
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-crm_phonecall_timesheet
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/117/12.0
+    :alt: Try me on Runbot
 
-=======================
-CRM Phonecall Timesheet
-=======================
+|badge1| |badge2| |badge3| |badge4| |badge5| 
 
 This module allows to generate timesheets from phone calls.
 
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+Go to ``Settings > Users`` and enable 'Show Scheduled Calls Menu'
+
 Usage
 =====
 
-* In phone calls tree, you can add an analytic account, and if the call has
-  any duration, a timesheet activity will be created.
-* You can choose how to input time: (1) set time you dedicated to call after
-  finishing it OR (2) create & save call with time set in 00:00 and Stop the
-  timer when call ends, this will set exactly the time you spent.
+1. Go to ``CRM > Phone Calls`` and create or edit a record
+2. Only records with project and duration > 0 will be generate an timesheet
 
-.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
-   :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/117/10.0
+** If you remove the project or change duration to 0 the timesheet will
+be deleted.
 
 Bug Tracker
```

Comment:
I think a bridge module would be needed in case of using `hr_timesheet_activity_begin_end` to get the right start and stop hours. You can note it here

Reply:
Added in ROADMAP.rst